wolfssl/wolfcrypt/aes.h

```diff
@@ -224,6 +228,11 @@ struct Aes {
                   GCM_NONCE_MID_SZ)];
 #endif
 #endif
+#if defined(WOLFSSL_KCAPI_AES)
+    struct kcapi_handle* handle;
+    int                  init;
+    word32 devKey[AES_MAX_KEY_SIZE/WOLFSSL_BIT_SIZE/sizeof(word32)];
```

Comment:
Does it make sense to combine `devKey` with the one right below? Then no odd issues if `WOLFSSL_KCAPI_AES` and `WOLF_CRYPTO_CB` are set.